packages/ui/package.json

```diff
@@ -22,13 +22,14 @@
     "@radix-ui/react-label": "^2.0.2",
     "@radix-ui/react-navigation-menu": "^1.1.4",
     "@radix-ui/react-radio-group": "^1.1.3",
+    "@radix-ui/react-select": "^1.2.2",
     "@radix-ui/react-switch": "^1.0.3",
     "@radix-ui/react-tabs": "^1.0.4",
-    "@radix-ui/react-select": "^1.2.2",
     "clsx": "^2.0.0",
     "cva": "npm:class-variance-authority@^0.7.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-phone-number-input": "^3.4.5",
```

Comment:
```suggestion
```
Just use `<input type="tel">`? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel

Reply:
We can also get phone-numbers from FEIDE, though we might need another scope

Reply:
1. If i'm a user who used FEIDE for my account creation, i would not like for the website to get my phone number and show it (this can be a toggle of course). I would rather want to be able to type it in myself, who knows, i might want to use a different phone number than the one registered at FEIDE.

2. By having the country code, not only does it look good in design, but also solves issues with possible members (exchange students) having non-norwegian phone numbers. 

I would like to keep it, but that's my opinion

Reply:
> also solves issues with possible members (exchange students) having non-norwegian phone numbers.

Which problems? This dependency seems quite unnecessary, if we wanted to validate that the input data is good we would actually to send a test-sms with a some kind of code akin to 2FA, but that costs money.

> If i'm a user who used FEIDE for my account creation, i would not like for the website to get my phone number and show it (this can be a toggle of course). I would rather want to be able to type it in myself, who knows, i might want to use a different phone number than the one registered at FEIDE.

I agree you might want to register your phone number as something different from FEIDE, but my issue was with adding a whole new dependency which seems unnecessary and just gives us technical debt. I think the feature is fine, just remove the extra JS

Reply:
Regardless of outcome this PR should remove the `@fadi-ui/react-country-flag` dependency